README.md

```diff
@@ -15,6 +15,30 @@
 pest is a [PEG](https://en.wikipedia.org/wiki/Parsing_expression_grammar) parser
 generator with *simplicity* and *speed* in mind.
 
+## Usage
+You will need Cargo and Rust, follow the [installation instructions](https://www.rust-lang.org/en-US/downloads.html) 
+to get them.
+
+Add the following to `Cargo.toml`:
+
+```toml
+pest = "0.4"
+```
+
+and in your Rust `lib.rs` or `main.rs`:
+
+```rust
+#[macro_use]
+extern crate pest;
+```
+
+Pest uses normal `Cargo` commands so you can build it by calling `cargo build` and run the test by calling `cargo test`.
+
+When you run the tests, you will see test names like `_0`, `_1` etc. Those are docstring tests.
```

Comment:
Remove this one too. The idea is to make "Elegant" more visible. The strongest point about pest is not it's ease of installation, but ease of use and elegance, and that needs to be visible.